manager/assets/modext/widgets/source/modx.panel.source.js

```diff
@@ -64,7 +64,7 @@ MODx.panel.Source = function(config) {
                                 ,anchor: '100%'
                                 ,listeners: {
                                     'keyup': {scope:this,fn:function(f,e) {
-                                        Ext.getCmp('modx-source-header').getEl().update(_('source')+': '+f.getValue());
+                                        Ext.getCmp('modx-source-header').getEl().update(f.getValue());
```

Comment:
Maybe here it is also worth outputting in the format `Ext.util.Format.htmlEncode(f.getValue())`? I do not know if it is necessary, but the different output variant `f.getValue()` is confusing.